src/ui/Templates/TemplateToHtml.ts

```diff
@@ -0,0 +1,71 @@
+import { IQueryResults } from '../../rest/QueryResults';
+import { IQueryResult } from '../../rest/QueryResult';
+import { Assert, QueryUtils, Component, Initialization, QueryStateModel } from '../../Core';
+import { IInitResult } from '../Base/Initialization';
+import { Template } from './Template';
+import { SearchInterface } from '../../ui/SearchInterface/SearchInterface';
+import { ResultList } from '../ResultList/ResultList';
+import { ValidLayout } from '../ResultLayoutSelector/ValidLayout';
+
+export interface ITemplateToHtml {
+  resultTemplate: Template;
+  queryStateModel: QueryStateModel;
+  searchInterface: SearchInterface;
+}
+
+export class TemplateToHtml {
```

Comment:
Ideally we'd add a couple of UT for this specific class.

However, I understand they are indirectly "tested" through the result list component, so I guess it's fine.

Reply:
Yeah, that is what I realized when I move everything because a lot was broken. This is why I didn't add any test